threedigrid_builder/grid/cross_section_definitions.py

```diff
@@ -229,10 +249,10 @@ def tabulate_tabulated(shape, width, height):
             f"(got: {height})."
         )
 
-    return shape, np.max(widths), np.max(heights), np.array([heights, widths]).T
+    return shape, np.max(widths), np.max(heights), np.array([heights, widths]).T, None
 
 
-def tabulate_yz(shape, width, height):
+def tabulate_yz(shape, width, height, friction_values, vegetation_stem_densities, vegetation_stem_diameters, vegetation_heights, vegetation_drag_coefficients):
```

Comment:
This call signature doesn't match the other call signatures (at line 68 you did adjust the altered return value, but the new parameters aren't provided)

I think it is good to leave the call signature as is and provide the additional vegentation coefs at a different moment.